client/my-sites/checkout/checkout-thank-you/jetpack-checkout-siteless-thank-you-completed.tsx

```diff
@@ -44,9 +44,10 @@ const JetpackCheckoutSitelessThankYouCompleted: FC< Props > = ( { productSlug }
 	return (
 		<Main wideLayout className="jetpack-checkout-siteless-thank-you-completed">
 			<PageViewTracker
+				options={ { useJetpackGoogleAnalytics: true } }
```

Comment:
Just as a recommendation, I think it would be a good idea to tell the whole team about this property. I wasn't aware of it and I believe I'm not the only one.